test/Microsoft.IdentityModel.Tokens.Tests/JsonWebKeySetTests.cs

```diff
@@ -417,6 +417,15 @@ public static TheoryData<JsonWebKeySetTheoryData> GetSigningKeysTheoryData
             }
         }
 
+        [Fact]
+        public void JsonDeserialize()
```

Comment:
I'm not really sure what can be the right strategy to run this test only if the TFM is `net8.0` and above, apart from putting a `#if NET8_0_OR_GREATER` around the test. I'm open to suggestions :)

Reply:
@joegoldman2 yes you can put NET_8_OR_GREATER

Reply:
Perfect, done!